index.js

```diff
@@ -1,92 +1,99 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
+console.log("%cQuestion 1", "color: red")
+let counter = 1;
+  counter = 2;
+console.log(counter)
 // // => 2
-// console.log("%c----------", "color: red") 
-
-
+console.log("%c----------", "color: red")
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
+// *** after testing, comment the line causing the error back in
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red")
+const name = "Da-mell"
+console.log(name)
 // => "Raffy"
-// name = "Not Raffy" 
+// name = "Not Raffy"
 // => TypeError
-// console.log("%c----------", "color: red") 
-
-
-
+console.log("%c----------", "color: red")
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
-
-// console.log(drinkWater(12))
+console.log("%cQuestion 3", "color: red")
+const drinkWater = (currentThirstLevel) => {
+  if (currentThirstLevel >= 12)
+  {
+    console.log("Man I sure am thirsty")
+     currentThirstLevel -= 1
+  }
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
```

Comment:
nice! I don't think you needed the constraint for only if the currentThirstLevel was >= 12 but it's a cool idea! Another way you can write line 27 is 

currentThirstLevel --